tfac/gene_module.py

```diff
@@ -16,7 +16,7 @@
 COLOR_CYCLE = plt.rcParams['axes.prop_cycle'].by_key()['color']
 PATH_HERE = dirname(abspath(__file__))
 working_dir = f'{PATH_HERE}/module_output'  # This is where Python will save output
-input_data = f'{PATH_HERE}/data/ohsu/MDD_RNAseq_Level4.txt'  # Path to the input file
+input_data = f'{PATH_HERE}/data/ohsu/MDD_RNAseq_Level4_fixed.txt'  # Path to the input file
```

Comment:
What is `MDD_RNAseq_Level4_fixed`? Why is it different from `MDD_RNAseq_Level4`?

Reply:
It is properly zscored. If I had kept the name, it was having conflicts..

Reply:
Just z-score on input. Otherwise, this violates the principal of retaining the original data on which the analysis is based.

Reply:
But the wgcna function directly accesses the data regardless of what we have done to it in the previous lines. Do you mean we should have both the zscored-data and the original files?

Reply:
Yes, WCGNA needs this, but that doesn't mean it needs to be stored in the repo. The z-scored data can be trivially derived from the original data file, but not vice versa. In fact, if later you want to filter the genes by expression level, the z-scored file lacks the necessary information to do so.

Reply:
I addressed this issue, I hope the edit I made is what you intended.
Also, removed genes with lots of zeros. I am letting it run now, it should be done in a couple of hours now that the size of the data is smaller.